test/cache_test.rb

```diff
@@ -250,8 +250,9 @@ def test_a_serializer_rendered_by_two_adapter_returns_differently_fetch_attribut
       # Assert attributes are serialized correctly
       serializable_alert = serializable(alert, serializer: AlertSerializer, adapter: :attributes)
       attributes_serialization = serializable_alert.as_json
-      assert_equal expected_fetch_attributes, alert.attributes
-      assert_equal alert.attributes, attributes_serialization
+      alert_attributes_symbolized = alert.attributes.symbolize_keys
```

Comment:
I can see the value of adding a test for symbolizing, but it looks like you're replacing the existing one?